### PR TITLE
support minimal (Linux) and full (Linux + Xen) imgs at the same time

### PR DESCRIPTION
**Pull request by macpijan**
support minimal (Linux) and full (Linux + Xen) imgs at the same time
    
Should fix: https://github.com/3mdeb/meta-trenchboot/issues/44
    
Renamed the images to better match their content/usage to name than
before.
    
Udpdated documentation to reflect the actual content of the layer.
    
For development, it's recommended to use the -full image.
    
The -minimal image might still be useful later on, for demonstration
purposes, or perhaps for Linux upstreaming process as well.
    
xen recipes must be moved to dynamic-layers, to allow for building
without meta-virtualization enabled.

### CODE REVIEW

**Comment by macpijan** [comment]
Both images built for me.

Still need testing on HW.

**Review by SergiiDmytruk — COMMENTED** [review]
Linux in the full image worked fine on hardware with and without DRTM.  The minimal image somehow failed to do DRTM.  Not sure why given the identical GRUB configuration for Linux.  Maybe something went wrong on my end.  Worth testing it.

**Comment by krystian-hebel** [comment]
On UEFI I'm getting:
```
error: can't find command `echo'.
```
instead of the normal output of `echo`. This was the case with previous version as well.

**Comment by macpijan** [comment]
> error: can't find command `echo'.

I can imagine https://github.com/3mdeb/meta-trenchboot/pull/49/commits/543a022c05a6bd3f56c3ef769a3132dd814e4849 might help here (not tested yet)

**Comment by macpijan** [comment]
I got an offline confirmation from @krystian-hebel that this has worked on HW, so I will go ahead and merge it if I manage to resolve the remaining suggestions.

File: README.md
[the remark this answers is not in the document]
Comment (macpijan) [reply README.md:1]: d384869a14ea623ff0b09068d1bfc92cc1c19177

**Pull request merged**